rclrs/src/wait.rs

```diff
@@ -35,16 +52,11 @@ impl Drop for rcl_wait_set_t {
 }
 
 impl WaitSet {
-    /// Creates and initializes a new WaitSet object.
-    pub fn new(
-        number_of_subscriptions: usize,
-        number_of_guard_conditions: usize,
-        number_of_timers: usize,
-        number_of_clients: usize,
-        number_of_services: usize,
-        number_of_events: usize,
-        context: &mut rcl_context_t,
-    ) -> Result<Self, RclReturnCode> {
+    /// Creates a new wait set.
+    ///
+    /// The given number of subscriptions is a capacity, corresponding to how often
+    /// [`WaitSet::add_subscription`] may be called.
+    pub fn new(number_of_subscriptions: usize, context: &Context) -> Result<Self, RclReturnCode> {
```

Comment:
Is the plan here to add back in the other parameters as we implement them?

Reply:
Yes. Or maybe do some kind of builder pattern.